dashboard/dashboard-impl/src/main/java/org/textadventure/dashboard/User.java

```diff
@@ -0,0 +1,79 @@
+package org.textadventure.dashboard;
+
+import java.lang.ref.SoftReference;
+
+/**
+ * Created by ������ on 08.12.2015.
+ */
+public class User {
+
+    private int id;
+    private String login;
+    private String userName;
+    private int age;
+    private String date;
+    private int questCount;
+    private float rating;
+
+    private String passwordHash;
+    private boolean banned;
+
+    public User(int id, String login, String userName, int age, String date, int questCount, float rating, String passwordHash, boolean banned){
```

Comment:
I know it is a class for test, but this constructor is very complicated.
You need use set-method or complex object for initialization